src/pf-tabs/index.html

```diff
@@ -6,48 +6,276 @@
   <link rel="stylesheet" href="../../dist/css/patternfly.css">
   <link rel="stylesheet" href="../../dist/css/patternfly-additions.css">
   <link rel="stylesheet" href="../../dist/css/patternfly-webcomponents.css">
+  <style>
+      .pf-tabrow-contents button {
+        margin-left: 5px;
+      }
+  </style>
 </head>
 
 <body>
   <div class="container">
     <div class="page-header">
       <h1>Tabs</h1>
     </div>
+
+    <pf-tabs>
+      <pf-tab content-id="example-one-tab-one">Tab One</pf-tab>
+      <pf-tab content-id="example-one-tab-two">Tab Two</pf-tab>
+    </pf-tabs>
+    <pf-tab-content content-id="example-one-tab-one">
+      <p> tab one content </p>
+    </pf-tab-content>
+    <pf-tab-content content-id="example-one-tab-two">
+      <p>tab two content </p>
+    </pf-tab-content>
+
+    <h3>Secondary Tabs</h3>
     <pf-tabs>
-      <pf-tab tab-title="Tab One" active="true">
-        <p>Tab one content here</p>
+      <pf-tab content-id="example-two-tab-one">Tab One</pf-tab>
+      <pf-tab content-id="example-two-tab-two">Tab Two</pf-tab>
+    </pf-tabs>
+    <pf-tab-content content-id="example-two-tab-one">
+      <pf-tabs tabs-class="nav nav-tabs nav-tabs-pf">
+        <pf-tab content-id="example-two-tab-one-secondary-one">Tab One Secondary Tab One</pf-tab>
+        <pf-tab content-id="example-two-tab-one-secondary-two">Tab One Secondary Tab Two</pf-tab>
+      </pf-tabs>
+      <pf-tab-content content-id="example-two-tab-one-secondary-one">
+        <p>tab one secondary tab one content</p>
+      </pf-tab-content>
+      <pf-tab-content content-id="example-two-tab-one-secondary-two">
+        <p>tab one secondary tab two content</p>
+      </pf-tab-content>
+    </pf-tab-content>
+    <pf-tab-content content-id="example-two-tab-two">
+      <p>tab two content </p>
+    </pf-tab-content>
+
+
+    <h3>Justified Tabs</h3>
+    <pf-tabs tabs-class="nav nav-tabs nav-justified">
+      <pf-tab content-id="example-three-tab-one">Tab One</pf-tab>
+      <pf-tab content-id="example-three-tab-two">Tab Two</pf-tab>
+      <pf-tab content-id="example-three-tab-three">Tab Three</pf-tab>
+    </pf-tabs>
+    <pf-tab-content content-id="example-three-tab-one">
+      <p> tab one content </p>
+    </pf-tab-content>
+    <pf-tab-content content-id="example-three-tab-two">
+      <p>tab two content </p>
+    </pf-tab-content>
+    <pf-tab-content content-id="example-three-tab-three">
+      <p>tab three content </p>
+    </pf-tab-content>
+
+    <h3>Tabs with Dropdowns</h3>
+    <pf-tabs>
+      <pf-tab content-id="example-four-tab-one">Tab One</pf-tab>
+      <pf-tab content-id="example-four-tab-two">Tab Two</pf-tab>
+      <pf-tab content-id="example-four-tab-three">
+        <pf-dropdown class="dropdown" id="dropdown1">
+          <a class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
I think attribute "data-***" is specific for bootstrap, it should not appear in patternfly-webcomponents